bigchaindb/backend/rethinkdb/config.py

```diff
@@ -0,0 +1,48 @@
+"""Database configuration functions."""
+import logging
+from functools import partial
+
+import rethinkdb as r
+
+from bigchaindb.backend import config
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+TABLES = ('bigchain', 'backlog', 'votes')
```

Comment:
Hmm, maybe put this in `backend/rethinkdb/__init__.py`? I know `rethinkdb/schema.py` also uses these table names in an iteration.